src/graphql/account.rs

```diff
@@ -464,6 +456,104 @@ impl AccountMutation {
     }
 }
 
+fn validate_password(account: &types::Account, username: &str, password: &str) -> Result<()> {
+    if !account.verify_password(password) {
+        info!("wrong password for {username}");
+        return Err("incorrect username or password".into());
+    }
+    Ok(())
+}
+
+fn validate_last_signin_time(account: &types::Account, username: &str) -> Result<()> {
```

Comment:
I would like to suggest using `validate_last_sign_in_time` instead of `validate_last_signin_time` for consistency. 

From what I googled online, I got this search result, so I am leaning to 'sign in', rather than `signin` or `sign-in`.

<img width="620" alt="image" src="https://github.com/user-attachments/assets/ad43f7ef-3225-430f-96fe-2c6c6c5e9230">